test/setup/DeviceServerMock.js

```diff
@@ -0,0 +1,11 @@
+// @flow
```

Comment:
Ok, sounds good.  Once you figure them it it will make writing tests so much easier.

Then you can write tests where the mock fails and make sure that our code is returning the correct response code.